tools/hardhat/hardhat.config.ts

```diff
@@ -0,0 +1,55 @@
+import { vars, HardhatUserConfig } from 'hardhat/config'
+import { ethers } from 'ethers'
+import '@nomicfoundation/hardhat-toolbox'
+// import precompile tasks
+import './tasks/validatorpool'
+import fs from 'fs'
+import path from 'path'
+
+const BUILD_DIR = '../../.localnet/'
+const COUNT = 4
+
+function getPrivKeys (): string[] {
+  const strings: string[] = vars.get('MEZO_ACCOUNTS', '').split(',')
+  const keys: string[] = []
+  if (strings[0] !== '') {
+    // Mezo accounts have been set already
+    for (const str of strings) {
+      if (str !== '') {
+        keys.push(str)
+      }
+    }
+  } else {
+    for (let i = 0; i < COUNT; i++) {
+      const filePath = path.resolve(`${BUILD_DIR}node${i}/mezod/key_seed.json`)
+      const seed = JSON.parse(fs.readFileSync(filePath, 'utf8'))
+      const pk: string = ethers.Wallet.fromPhrase(seed.secret).privateKey
+      keys.push(pk)
```

Comment:
Imoh, this part can be removed since this hardhat configuration should support passing accounts only via `npx hardhat vars set MEZO_ACCOUNTS`

Reply:
Agree [c10ca8b](https://github.com/mezo-org/validator-kit/pull/1/commits/c10ca8b4ad230a64b1f5702c4fe844bea8072512)